aten/src/ATen/detail/HIPHooksInterface.h

```diff
@@ -20,7 +22,7 @@ namespace at {
 // which we may want to call into from CPU code (and thus must be dynamically
 // dispatched, to allow for separate compilation of HIP code).  See
 // CUDAHooksInterface for more detailed motivation.
-struct TORCH_API HIPHooksInterface {
+struct TORCH_API HIPHooksInterface : AcceleratorHooksInterface {
```

Comment:
FYI @jeffdaily this makes HIP an "accelerator". I'm still not sure if you use it but that's just enabling more device-generic feature for the HIP device so I guess you're happy with it. We can remove it if you are not!

Reply:
IIRC we don't use HIPHooksInterface but rather a hipified version of the CUDAHooksInterface.  In any case, we're okay with being an Accelerator.  Is there an RFC or something similar describing pytorch's move to these generic interfaces?

Reply:
The short version is described in https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/DeviceAccelerator.h